README.md

```diff
@@ -124,7 +124,7 @@ $ docker run -dP --name="shib-local-test" org_id/docker-shibboleth-idp
 From source:
 
 ```
-$ docker build --tag="org_id/shibboleth-idp" github.com/jtgasper3/docker-shibboleth-idp
```

Comment:
This is building from github.com and therefore needs the `docker-` prefix. Please change that and I'll merge the PR in.